fsharp-backend/src/ApiServer/Api/APIOps.fs

```diff
@@ -43,6 +42,10 @@ let empty : Op.AddOpResult =
 
 let causesAnyChanges (ops : PT.Oplist) : bool = List.any Op.hasEffect ops
 
+/// API endpoint to add a set of Operations in a Canvas
```

Comment:
These are properly called Ops, not Operations. Might be worth defining in Op.fs that an Op is an Operation, but I wouldn't use the full word here when we only use an abbreviation throughout the code.